src/main/java/com/doit/api/user/UserService.java

```diff
@@ -10,56 +15,66 @@
 
 
 @Service
-public class UserService {
+@AllArgsConstructor
+public class UserService implements UserDetailsService {
 
     @Autowired
-    private PasswordEncoder passwordEncoder;
-
+    private final BCryptPasswordEncoder bCryptPasswordEncoder;
     @Autowired
-    private UserRepository userRepository;
+    private final UserRepository userRepository;
+
+    public final static String USER_NOT_FOUND_MSG = "user with email %s not found";
```

Comment:
If this is only used once, we don't need to put it in a variable.

Reply:
Done.